webgui/scripts/codecompass/viewHandler.js

```diff
@@ -31,6 +31,7 @@ function (lang, Deferred, model) {
       InfoTree               :  6, /** Info tree item (eg. CppInforTree)**/
       FileManagerContextMenu :  7, /** File manager context menu (eg. Diagram directory)**/
       InfoPage               :  8, /** Info page (eg. Credit, User Guide) **/
+      MicroserviceContextMenu:  9  /** Microservice navigator context menu **/
```

Comment:
Again, `helm` vs. `yaml` vs. `microservice`. Three different naming for the same thing. Confusing.

Reply:
In this case, I would leave it like this. The primary purpose of the Helm chart analysis is to identify microservices and their connections, so I think this naming is correct here. I'm unifying it in other places.

Reply:
As far as I understand this plugin is capable for Kubernetes microservice analysis based on Helm charts. Naming it simply microservice analysis in general is a bit like naming the "C++ plugin" as "Programming language plugin", as that would be a much wider concept.

There are other microservice orchestration frameworks beside Kubernetes, like Docker Swarm, Helios, Amazon ECS or Mesos, just to name a few. This plugin does not deal with those systems, instead it is fundamentally coupled with Helm charts for Kubernetes. Thus, in my opinion, the naming should use the `helm` keyword mostly and also towards the end user it should be named something like that.